DOCS-58 Introduction (#570)
streamr-dev/streamr-platform: app/src/newdocs/content/introduction.mdx

        <Link className={styles.link} to={formatPath(links.newdocs.tutorials)}>
            Jump to section
            <SvgIcon name="back" className={styles.forwardArrow} />
        </Link>
    </Panel>
</div>

<hr />

## What is Streamr?

Streamr is an open source, crowdfunded, decentralized platform for realtime data. At its heart is the <Link to={links.site.network}>Streamr Network</Link>, which transports <Link to={links.newdocs.streams}>streams</Link> of messages from data publishers to subscribers, appearing to the user as a global publish/subscribe messaging service.

It is similar in functionality to message queues, enterprise messaging systems, MQTT brokers, Apache Kafka, or pub/sub cloud messaging services. The key features of such systems, including Streamr, are:

- Publishing messages in order to make them available to subscribers
- Subscribing to messages in order to process them as they occur
- Storing messages for later use

You can interact with Streamr using the <Link to={links.newdocs.core}>Core</Link> user interface, <Link to={links.newdocs.api}>SDKs</Link>, or directly via the <Link to={links.newdocs.api}>API</Link>.

> In addition to the Network, the Streamr stack includes a <Link to={links.newdocs.marketplace}>Marketplace</Link> for data sharing and monetization, and <Link to={links.newdocs.canvases}>Canvases</Link> for data processing and analytics as part of the Core webapp.

Reviewer: ```suggestion
In addition to the Network, the Streamr stack includes a <Link to={links.newdocs.marketplace}>Marketplace</Link> for data sharing and monetization, and <Link to={links.newdocs.canvases}>Canvases</Link> for data processing and analytics as part of the Core web app.
```

> Using the built-in tooling is optional, as Streamr also integrates with many popular data frameworks including <Link to={links.blogPosts.spark}>Apache Spark</Link> and <Link to={links.blogPosts.nodered}>Node-RED</Link>.

Reviewer: ```suggestion
Using the built-in tooling is optional, as Streamr also integrates with many popular data frameworks including <a target="_blank" href={links.blogPosts.spark}>Apache Spark</a>and <a target="_blank" href={links.blogPosts.nodered}>Node-RED</a>.
```

To start using Streamr right away, check out the <Link to={links.newdocs.gettingStarted}>Getting Started</Link> section and the <Link to={links.newdocs.tutorials}>Tutorials.</Link>

## Why use Streamr?

The big difference to comparable data frameworks is that Streamr aims to be *decentralized* and *peer-to-peer*. There are plenty of advantages to decentralization:

- You don't need to set up any servers to use the service
- You don't need to trust any third party with your data
- You benefit from the network effects of everyone participating in the same network, instead of having to operate in an isolated silo

> Instead of a centralized party, the Network is run by its community of users, and it heavily relies on cryptography to remove the need for trust. This creates a permissionless and neutral network for realtime data, which is highly suitable for use cases such as data sharing, data monetisation, and the <Link to={links.external.decentralizedWeb}>decentralised web</Link>, extending the capabilities of the underlying internet without compromising its open and decentralised nature.

Reviewer: ```suggestion
Instead of a centralized party, the Network is run by its community of users, and it heavily relies on cryptography to remove the need for trust. This creates a permissionless and neutral network for realtime data, which is highly suitable for use cases such as data sharing, data monetisation, and the <a target="_blank" href={links.external.decentralizedWeb}>decentralised web</a>, extending the capabilities of the underlying internet without compromising its open and decentralised nature.
```

> Another major difference is that Streamr utilizes a companion blockchain, currently <Link to={links.external.ethereum}>Ethereum</Link>. The blockchain provides users with secure digital identities, enables direct transfers of value in the form of a Streamr-specific cryptocurrency <Link to={links.newdocs.dataToken}>DATA</Link>, and provides shared ground truth for the network nodes.

Reviewer: ```suggestion
Another major difference is that Streamr utilizes a companion blockchain, currently <a target="_blank" href={links.external.ethereum}>Ethereum</a>. The blockchain provides users with secure digital identities, enables direct transfers of value in the form of a Streamr-specific cryptocurrency <Link to={links.newdocs.dataToken}>DATA</Link>, and provides shared ground truth for the network nodes.
```

> Another major difference is that Streamr utilizes a companion blockchain, currently <a target="_blank" href={links.external.ethereum}>Ethereum</a>. The blockchain provides users with secure digital identities, enables direct transfers of value in the form of a Streamr-specific cryptocurrency <Link to={links.newdocs.dataToken}>DATA</Link>, and provides shared ground truth for the network nodes.

Reviewer: ```suggestion
Another major difference is that Streamr utilizes a companion blockchain, currently <a target="_blank" href={links.external.ethereum}>Ethereum</a>. The blockchain provides users with secure digital identities, enables direct transfers of value in the form of a Streamr-specific cryptocurrency <Link to={links.newdocs.dataToken}>DATA</Link>, and provides a shared ground truth for the network nodes.
```

## Work in Progress

> While Streamr already works today, it is not yet feature-complete or fully decentralized. It is being very actively developed towards decentralization, node incentives, and new features. Learn more about the roadmap on the <Link to={links.streamrSite}>project website</Link>.

Reviewer: ```suggestion
While Streamr already works today, it is not yet feature-complete or fully decentralized. It is being very actively developed towards decentralization, node incentives, and new features. Learn more about the roadmap on the <a target="_blank" href={process.env.PLATFORM_ORIGIN_URL}>project website</a>.
```

        <Link className={styles.link} to={formatPath(links.newdocs.canvases)}>
            Learn more
            <SvgIcon name="back" className={styles.forwardArrow} />
        </Link>
    </Panel>
</div>

<div className={styles.panels}>
    <Panel>
        <h5>Dashboards</h5>
        <p>Dashboards are collections of realtime visualisations, cherry-picked from your canvases, presented nicely without all the wiring.</p>
        <Link className={styles.link} to={formatPath(links.newdocs.dashboards)}>
            Learn more
            <SvgIcon name="back" className={styles.forwardArrow} />
        </Link>
    </Panel>
    <Panel >
        <h5>Products</h5>
        <p>Products are collections of streams which can be published onto the Marketplace and purchased by other Streamr users.</p>
        <Link className={styles.link} to={formatPath(links.newdocs.products)}>
            Learn more
            <SvgIcon name="back" className={styles.forwardArrow} />
        </Link>
    </Panel>
</div>

## Getting Help

On the below channels, you can find others developing on Streamr, as well as get in touch with the Streamr core contributors:

>  - <Link to={links.community.devForum}>Community dev forum</Link>

Reviewer: ```suggestion
- <a target="_blank" href={links.community.devForum}>Community dev forum</a>
```

>  - <Link to={links.community.telegram}>Telegram</Link>

Reviewer: ```suggestion
- <a target="_blank" href={links.community.telegram}>Telegram</a>
```

>  - <Link to={links.community.reddit}>Reddit</Link>

Reviewer: ```suggestion
- <a target="_blank" href={links.community.reddit}>Reddit</a>
```

## Digging Deeper

> To understand the exact inner workings of Streamr, and perhaps help build it, check the <Link to={links.newdocs.technicalNotes}>Technical Notes section</Link> and of course the <Link to={links.community.github}>project GitHub</Link>.

Reviewer: ```suggestion
To understand the exact inner workings of Streamr, and perhaps help build it, check the <Link to={links.newdocs.technicalNotes}>Technical Notes section</Link> and of course the <a target="_blank" href={links.community.github}>project GitHub</a>.
```
